introduction.md

```diff
@@ -5,3 +5,9 @@ I'm lucas Romano, 32 years old web developer from Brussels, Belgium
 
 ## prefered path ##
 JavaScript
+
+## Deadly Skills ##
+* JavaScript
+* Ruby
+
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?